lib/elixir/lib/regex.ex

```diff
@@ -818,7 +818,30 @@ defmodule Regex do
     |> IO.iodata_to_binary()
   end
 
-  @escapable '.^$*+?()[]{}|#-\\\t\n\v\f\r\s'
+  @escapable [
```

Comment:
If we can't use sigils here, one alternative is: `String.to_charlist("...")`

Reply:
`String` causes a bootstrapping error too, but `:binary.bin_to_list(".^$*+?()[]{}|#-\\\t\n\v\f\r\s")` works 👍